mappings/net/minecraft/client/gui/EditBox.mapping

```diff
@@ -0,0 +1,169 @@
+CLASS net/minecraft/class_7530 net/minecraft/client/gui/EditBox
+	COMMENT A multiline edit box with support for basic keyboard shortcuts.
+	COMMENT This should not be used directly; {@link net.minecraft.client.gui.widget.EditBoxWidget}
+	COMMENT provides the GUI for the edit box.
+	FIELD field_39511 UNLIMITED_LENGTH I
+		COMMENT A constant denoting that the edit box accepts unlimited amount of text. Edit box
+		COMMENT widgets with such edit boxes do not show the current text length indicator.
+	FIELD field_39512 CURSOR_WIDTH I
+	FIELD field_39513 textRenderer Lnet/minecraft/class_327;
+	FIELD field_39514 lines Ljava/util/List;
+	FIELD field_39515 text Ljava/lang/String;
+	FIELD field_39516 cursor I
+	FIELD field_39517 selectionEnd I
+	FIELD field_39518 selecting Z
+	FIELD field_39519 maxLength I
+	FIELD field_39520 width I
+	FIELD field_39521 changeCallback Ljava/util/function/Consumer;
+	FIELD field_39522 cursorChangeCallback Ljava/lang/Runnable;
```

Comment:
One note: Yarn typically uses "listener" for these, I couldn't find any uses of "callback" via Github search. Might be good to change for consistency?

Reply:
Hmm, `callback` and `listener` are both used, but I think `listener` also works.